man/bundle-config.ronn

```diff
@@ -138,6 +138,8 @@ learn more about their operation in [bundle install(1)][bundle-install].
    `bundle install`.
 * `console` (`BUNDLE_CONSOLE`):
    The console that `bundle console` starts. Defaults to `irb`.
+* `deployment` (`BUNDLE_DEPLOYMENT`):
+   Disallow changes to the `Gemfile`.
```

Comment:
I think this wording is a little unclear... how do you feel about something like "Block commands if the Gemfile has been changed and no longer matches the lock."